src/qt/forms/optionsdialog.ui

```diff
@@ -86,6 +86,16 @@
          </property>
         </widget>
        </item>
+       <item>
+        <widget class="QCheckBox" name="gridcoinAtStartupminimised">
```

Comment:
this is not proper camelCase

Reply:
would gridcoinAtStartupMinimised be proper?